model/registry/incident.yaml

```diff
@@ -0,0 +1,25 @@
+groups:
+  - id: registry.incident
+    prefix: incident
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to incidents.
```

Comment:
the incident is a very wide term that could mean all different source of incidents which might need different attributes sets under them. Can we be more specific here - is it security incident, alert, etc, and pick a corresponding name?

Reply:
+1 to this comment. Incidents are a very broad category, and there are a lot of different vendors with different data models of incidents (and notifications, pages, etc.). Some examples include:

  - Google Cloud alerts: https://cloud.google.com/monitoring/alerts
 -  AWS Cloud watch alarms: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html
 - JIRA Ops incidents: https://www.atlassian.com/software/jira/service-management/product-guide/getting-started/incident-management#how-it-works
 - etc.

... among others. Given the breadth and implications, I wonder if it might make sense to break this area out into a separate pull request in its own right.

Reply:
Can definitely break incident out of this pull request. 

Fundamentally I'm referring to an incident here that causes an outage. Incidents are necessary for the calculate of DORA Metrics. The breadth of different systems that emit incidents is certainly wide, which is why I left this with only a couple key attributes that I think would be able to map to all and provide the context needed alongside other attributes to make the proper relations. 

So perhaps this should be `outage.incident` or just simply removed & put in another pull request. Either option is fine with me.

Reply:
Opting to move this into a separate pull request for a better, more targeted discussion & iteration.